config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -124,27 +124,28 @@ find "$base" -name '*.yaml' -exec grep -q 'generated by gen-jobs.sh' '{}' \; -de
 # Resource usage of a job depends on whether it needs to build Kubernetes or not.
 resources_for_kubernetes () {
     local kubernetes="$1"
-
+    local indentation="$2"
+    local IFS=""
     # "master" is always built from source. The experimental version might be built from
     # source, depending on what is in the prow.sh script. Here we play it safe and
     # request more resource when testing the experimental version, just in case.
     case $kubernetes in master|$experimental_k8s_version|release-*) cat <<EOF
-      resources:
-        requests:
-          # these are both a bit below peak usage during build
-          # this is mostly for building kubernetes
-          memory: "9000Mi"
-          # during the tests more like 3-20m is used
-          cpu: 2000m
+    resources:
```

Comment:
Now that `resources` is in the body of the text, it is visible that it is already indented. That makes `$indention` more like `$additional_indention`.

I think it is cleaner to let `resources` start in the first column (i.e. no indention) and then choose the amount of spaces entirely where `resources_for_kubernetes` is called.

Reply:
that makes complete sense ... on it!

Reply:
Done!